py/tests/release.py

```diff
@@ -60,6 +61,9 @@
 chain:
     persist: true
     db_path: ./my_db
+
+fork_management:
+    network_id: ae_uat
```

Comment:
Why not using some other id? I guess we'll add the peers for this ID at some point and the test will take longer because it will connect to the real testnet.